src/app/team/locationForm/number/LocationNumberView.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link, withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { deletePhone, getLocation } from '../../../../actions';
+import { getPhoneNumbers } from '../../../../selectors/location';
+
+import Button from '../../../../components/button';
+import ConfirmationOptions from '../../../../components/form/ConfirmationOptions';
+import ConfirmationModal from './ConfirmationModal';
+
+import deleteIcon from './DeleteIcon.svg';
+
+const formatPhoneNumber = phone =>
+  `${phone.number.replace(/\./g, '-')}${
+    phone.extension ? ` ext. ${phone.extension}` : ''
+  }`;
+
+class LocationNumberView extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isEditing: false,
+      showDeleteModal: false,
+    };
+
+    this.toggleEdit = this.toggleEdit.bind(this);
+    this.onConfirmDelete = this.onConfirmDelete.bind(this);
+    this.onCancelDelete = this.onCancelDelete.bind(this);
+    this.deletePhone = this.deletePhone.bind(this);
+  }
+
+  async onConfirmDelete() {
+    const { showDeleteModal: { id } } = this.state;
+
+    await this.props.deletePhone(id);
+
+    this.setState({
+      showDeleteModal: false,
+    }, () => {
+      this.props.fetchResourceData();
+    });
+  }
+
+  onCancelDelete() {
+    this.setState({
+      showDeleteModal: false,
+    });
+  }
+
+  deletePhone(phone) {
+    this.setState({
+      showDeleteModal: phone,
+    });
+  }
+
+  toggleEdit() {
+    this.setState({
+      isEditing: !this.state.isEditing,
+    });
+  }
+
+  render() {
```

Comment:
This function (and file in general) are really clean and easy to follow.
This is great.